doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -352,6 +352,12 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-6-1 v2-6-0
+
+KRKNWK-18673: Bridged light bulb device report failure at reading and/or writing some of the onoff cluster's attributes.
```

Comment:
```suggestion
KRKNWK-18673: Bridged Light Bulb device sometimes reports a failure when reading or writing cluster attributes
  This can happen when reading or writing some of the attributes of the OnOff cluster.
```
Or something like this. The issue definition requires at least one line of description in the body text so it builds correctly.